locales/en-US/server.ftl

```diff
@@ -9,6 +9,7 @@ gNoShots
 gScreenshotsDescription = Screenshots made simple. Take, save, and share screenshots without leaving Firefox.
 gSettings = Settings
 gSignIn = Sign In
+gBannerMessage = Sign in or sign up to sync shots across devices, save your favorite shots forever.
```

Comment:
Change this to: 

Sign in or sign up to sync shots across devices and save your favorite shots forever.